src/test/regress/expected/failure_create_table.out

```diff
@@ -0,0 +1,597 @@
+CREATE SCHEMA failure_create_table;
+SET search_path TO 'failure_create_table';
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+-- Create distributed table in 1PC and 2PC,
+-- then check if the transaction can be rollbacked.
+SET citus.shard_replication_factor TO 1;
+SET citus.shard_count to 4;
+CREATE TABLE test_table(id int, value_1 int);
+-- Kill connection before sending query to the worker 
+SELECT citus.mitmproxy('conn.kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT create_distributed_table('test_table','id');
+ERROR:  connection error: localhost:57640
+DETAIL:  server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+SELECT count(*) FROM pg_dist_shard;
+ count 
+-------
+     0
+(1 row)
+
+\c - - - :worker_1_port
+SELECT count(*) FROM pg_class WHERE relname LIKE 'test_table%';
+ count 
+-------
+     0
+(1 row)
+
+\c - - - :master_port
+SET search_path TO 'failure_create_table';
+-- kill as soon as the coordinator sends CREATE SCHEMA
+SELECT citus.mitmproxy('conn.onQuery(query="^CREATE SCHEMA").kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT create_distributed_table('test_table', 'id');
+ERROR:  server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+CONTEXT:  while executing command on localhost:57640
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT count(*) FROM pg_dist_shard;
+ count 
+-------
+     0
+(1 row)
+
+SELECT run_command_on_workers($$SELECT count(*) FROM information_schema.schemata WHERE schema_name = 'failure_create_table'$$);
```

Comment:
It's really strange that the `CREATE SCHEMA` isn't created inside a transaction, so the other node is given a schema even though we aren't. I'm not sure, maybe add a comment saying this is expected behavior?